### PR TITLE
Create and Delete mutations for a user's activities

### PR DESCRIPTION
**Pull request by rer7891**
### Sidebar Checklist

- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project

### Issues Resolved
Resolves #

### Problem Addressed
Don't have a way for a user to pick categories and the BE to then associate those activities to their account. If a user doesn't like an activity there is no way to delete that from their account. 
### Solution Implemented
Creates a create_user_activities and delete_user_activity mutation query for the front end to send to the BE. 
### Other Notes
Added testing and added queries to the docs.

### CODE REVIEW

File: spec/graphql/mutations/users/create_user_activities_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        activity2 = Activity.create(name: "Bike Ride", est_time: "01:15")
+        activity3 = Activity.create(name: "Facial", est_time: "01:15")
+        activity4 = Activity.create(name: "Pedicure", est_time: "01:15")
+        category.activities << [activity1, activity2]
+        category.activities << [activity3, activity4]
+
+
+  			result = TreatYoSelfSchema.execute(query).as_json
+        activities = [activity1.name, activity2.name, activity3.name, activity4.name]
+        expect(result["data"]["user"]["activities"]).to eq(activities)
```
Comment (tyladevon) [inline spec/graphql/mutations/users/create_user_activities_spec.rb:23]: Looking at this test, I am finding it unclear as to how the activities are being connected to the user? Can we go over this today?

File: app/graphql/mutations/users/delete_user_activity.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module Mutations
+  module Users
+    class DeleteUserActivity < ::Mutations::BaseMutation
```
Comment (wrenisit) [inline app/graphql/mutations/users/delete_user_activity.rb:3]: This is a great fix for the issue. Thank you for making this happen.

File: spec/graphql/mutations/users/create_user_activities_spec.rb (hunk 1 above)
Comment (wrenisit) [reply spec/graphql/mutations/users/create_user_activities_spec.rb:23]: Also, can we add a 5th activity? That way we can ensure it's only adding what we tell it to.

Comment (rer7891) [reply spec/graphql/mutations/users/create_user_activities_spec.rb:23]: The activities are being connected to the user via the actual query. So in the resolver for create_user_activities the connection is made

Comment (tyladevon) [reply spec/graphql/mutations/users/create_user_activities_spec.rb:23]: Thanks Becky, that makes sense!

**Pull request merged**